### PR TITLE
Add size() helper for Auto*Array

### LINKED ISSUE #278: Consider adding AutoArray.size
As it already contains the Java array reference and JNIEnv, it could also support size, so that the users don't have to request it separately with `get_array_length`.

### PR DESCRIPTION
**Pull request by maurolacy**
## Overview

Closes #278. 

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by [automated tests](https://github.com/jni-rs/jni-rs/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/jni-rs/jni-rs/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has documentation
- [x] User-visible changes are mentioned in the Changelog
- [x] The continuous integration build passes

### CODE REVIEW

**Comment by maurolacy** [comment]
Tests are failing because of the commit() bug. Maybe we should disable that test in the meantime.

File: tests/jni_api.rs [hunk 1]
```diff
@@ -426,7 +432,7 @@ pub fn get_long_array_elements() {
     assert_eq!(res[2], 4);
 }
 
-#[test]
+//#[test]
```
Comment (stanislav-tkach) [inline tests/jni_api.rs:435]: I suppose marking test as ignored (`#[ignore]`) would be better. Ideally an explaining comment should be added too.

**Review by stanislav-tkach — APPROVED** [review]

File: tests/jni_api.rs (hunk 1 above)
Comment (maurolacy) [reply tests/jni_api.rs:435]: Sure. I didn't knew about `#[ignore]`, thanks.

**Review by dmitry-timofeev — APPROVED** [review]
👍

**Pull request merged**